content/docs/studio/rest-api.md

```diff
@@ -0,0 +1,58 @@
+# REST API
+
+Studio REST API is WIP and will be extended. The API is hosted under the `/api`
+route on the Studio server: https://studio.iterative.ai/api or
+https://your-domain/api in case of
+[self-hosted Studio](/doc/studio/self-hosting/installation).
+
+To use API, you need to generate
+[Studio access token](/doc/studio/user-guide/account-management#studio-access-token).
+
+## Download model
+
+Get signed url to download the model binary. Requires the model to be stored
+with DVC with s3 or azure remote. `path` in model annotation must point to a
+single file and not to a directory. Note, that you need to
+[set up remote cloud credentials](/doc/studio/user-guide/account-management#cloud-credentials)
+for Studio have rights to sign urls.
+
+```yaml
+Endpoint: api/model-registry/get-download-uris
+HTTP Method: GET
+```
+
+### Request
+
+```yaml
```

Comment:
Minor, but I find this format confusing. By combining it all into one yaml, it at first looks like `repo.desc` is one parameter instead of a description of the `repo` parameter.

Reply:
Thanks - changed. Looks better now?